packages/commands/src/withdraw-tokens.mjs

```diff
@@ -0,0 +1,133 @@
+import { Connection, Keypair } from '@solana/web3.js'
+import BN from 'bn.js'
+import chalk from 'chalk'
+import { program, Option } from 'commander'
+
+import { initializeAudiusLibs } from './utils.mjs'
+
+import { createAssociatedTokenAccountInstruction } from '@solana/spl-token'
+import {
+  Transaction,
+  sendAndConfirmTransaction,
+  PublicKey
+} from '@solana/web3.js'
+
+program
+  .command('withdraw-tokens')
+  .description('Send USDC from a user bank to an external address')
+  .argument('<account>', 'The solana address of the recipient')
+  .argument('<amount>', 'The amount of tokens to tip (in wei)')
```

Comment:
is this wording copied over from somewhere? don't think we're tipping here right?

Reply:
Oh yep. This was copied and modified from a different command. Will update.